FourSquare/Controllers/DetailVenue/ViewPageImageHeader/PageImageHeaderView.swift

```diff
@@ -17,18 +17,19 @@ class PageImageHeaderView: UITableViewHeaderFooterView {
     @IBOutlet weak var imagesPageControl: UIPageControl!
     var imagePageViewController: UIPageViewController?
 
-    var imageNames = ["detail_venue_image"] {
+    var photos: [Photo] = [] {
         didSet {
-            self.afterPageButton.hidden = self.imageNames.count == 1
-            self.imagesPageControl.numberOfPages = self.imageNames.count
+            self.afterPageButton.hidden = self.photos.count == 1
```

Comment:
in case of having no image name. afterPageButton still appears.
self.afterPageButton.hidden = self.imageNames.count <= 1